### DISCUSSION TITLE
Correct front matter for Computer Science doctoral thesis

### DISCUSSION THREAD

**Discussion by bernhardmgruber** [tud-cd/tudscr#92] [Q&A] [answered]
Hi! I am completing my doctoral thesis in Computer Science and wonder how to populate the front matter correctly.

My supervising professor is Prof. Wolfgang Nagel. Based on the organization of the TU Dresen Faculity of Computer Science [here](https://tu-dresden.de/ing/informatik/die-fakultaet/institute-und-professuren-1?set_language=en), and [this](https://tu-dresden.de/zih) about the ZIH (Center for information services and high performance computing), I came up with the following:
```latex
\faculty{Faculty of Computer Science}
\department{Center for information services and high performance computing (ZIH)}
\institute{Institute of Computer Engineering}
\chair{Chair of Computer Architecture}
```
On the ZIH website, it calls itself a department, but I am not sure whether this is the kind of department that is meant here.

Furthermore, the rendering requires almost two lines on the front page:
![image](https://github.com/tud-cd/tudscr/assets/1224051/951f1213-86b3-4fb5-bf8e-1727299be809)

So my question is, is the information I gather correct and have I correctly filled in the latex template?

Thank you for the help!

**Comment by mrpiggi** [comment]
This is probably something you should rather ask at your faculty as there are probably some formal requirements which I do not know.

**Reply by bernhardmgruber** [reply]
Alright. I thought since this is the official latex template, you would probably know how it should work :) But I can try and ask someone else. Maybe someone also sees this post here and can answer. Anyway, thx for replying!

**Reply by keinstein** [reply, marked as answer]
The ZIH is a department that does not belong to any faculty. So you can abuse `\faculty` like
```
\faculty{Center for information services and high performance computing (ZIH)}
```
Your example seems that you submit the thesis to the faculty of computer science. Then, you should just omit the `\department`

Maybe you want to add something like “In cooperation with Center for information services and high performance computing (ZIH)”. 

Please keep in mind that formal requirements vary from faculty to faculty and their enforcement can vary even between two groups in the same institute. So your near colleagues are the first choice for answering such questions.

**Reply by mrpiggi** [reply]
> Alright. I thought since this is the official latex template, you would probably know how it should work :)

Well, the classes are not official at all but where developed in my spare time.

Obviously, every structural unit at the university comes with its own individual requirements and formal guidelines, many of them without any reason and even resulting in bad typography. Certainly, you faced the insistence on 1.5 line spacing several times, didn't you?

Eventually, although there is an official manual on corporate design provided by the university, it is actuarial not enforced. And I can't be aware of every special treatment.
